### PR TITLE
Search stuff

### LINKED ISSUE #430: Article: All about searching
Need an article to go over the `SoftLayer_Search` service

- basic search
- advanced seaarch
- result ordering with an objectFilter should be possible

### PR DESCRIPTION
**Pull request by allmightyspiff**
Fixes #430 
Adds an article that explain how the SoftLayer_Search service works

### CODE REVIEW

File: content/article/softlayer_search.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+The `name` property referes to the type of SoftLayer Datatype being tracked, and the `properties` are the searchable fields, which are relational properties on the named datatype.
+
+# [SoftLayer_Search::search](#search_search) {#search_search .anchor-link}
+
+[SoftLayer_Search::search(search_term)](/reference/services/SoftLayer_Search/search/) takes in one parameter, and that is a string representing what you want to search for. The documentation here has a nice rundown of how to build out the API call. Basically you need to define which object types you want to search in, and what terms you want to search in those types for.
+
+## Find all Virtual and Hardware servers.
+
+By specifying an `_objectType` of `SoftLayer_Hardware` and ``SoftLayer_Virtual_Guest` and not defining a search term, this API call will return all Hardware and Virtual objects on your account.
```
Comment (ATGE) [inline content/article/softlayer_search.md:75]:
```suggestion
By specifying an `_objectType` of `SoftLayer_Hardware` and `SoftLayer_Virtual_Guest` and not defining a search term, this API call will return all Hardware and Virtual objects on your account.

```
looks like the is an additional single quote mark on `SoftLayer_Virtual_Guest`

**Review by ATGE — COMMENTED** [review]
✔️Tested view

File: content/article/softlayer_search.md (hunk 1 above)
Comment (allmightyspiff) [reply content/article/softlayer_search.md]: Thanks for catching that.

**Review by ATGE — APPROVED** [review]
looks good

**Pull request merged**